networking/advertise-service-ips.md

```diff
@@ -70,24 +84,25 @@ If your {{site.prodname}} deployment is configured to peer with BGP routers outs
 1. Check to see if you have a default BGPConfiguration.
 
    ```bash
-   calicoctl get bgpconfig default
+   kubectl get bgpconfiguration.projectcalico.org default
```

Comment:
I think everything from here downward isn't relevant to this PR and we should remove it.

Reply:
Are you saying to prune this topic to only one task: #### Advertise service cluster IP addresses? 

Or did you mean to remove all other steps after step 1 in the section, #### Advertise service cluster IP addresses?

Reply:
I meant that all of the changes in this PR from this point on are not correct for OSS Calico, and also aren't required so far as clarifying source IP preservation are concerned, so we shouldn't change anything here.

Reply:
Fixed.

Reply:
This line should still be reverted as well - it should say "calicoctl get bgpconfig default"

Reply:
Fixed.